src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -144,6 +117,15 @@ private void renumberColumns() {
 			c.setPos(i++);
 		}
 	}
+
+	public void setDefaultColumnEncodings() {
```

Comment:
Oh hell :/ I was hoping you would go the other way and simplify code. But,... reality.